evm_arithmetization/src/cpu/kernel/asm/core/precompiles/bn_add.asm

```diff
@@ -60,4 +56,6 @@ bn_add_return:
     %stack (parent_ctx, y) -> (parent_ctx, @SEGMENT_RETURNDATA, 32, y)
```

Comment:
Actually not even, because the offset between x and y writes is 32, and the address is already incremented after writing `x` so we don't need to pop and reconstruct the new address as we already have it, i.e. 

```asm
    %stack (parent_ctx, x, y) -> (parent_ctx, @SEGMENT_RETURNDATA, x, y)
    %build_address_no_offset
    // stack: addr_x, x, y, base_addr, kexit_info
    MSTORE_32BYTES_32
    // stack: addr_y = addr_x + 32, y, base_addr, kexit_info
    MSTORE_32BYTES_32
    // stack: addr, base_addr, kexit_info
```

Thanks for catching it!